File: year2020/month10/day1002/ms_1002.cpp
#include <iostream>

using namespace std;

void recursionPrint(int n);

int main() {
	int n;

	cin >> n;

	if (0 < n && n <= 50)
		recursionPrint(n);

	return 0;
}

void recursionPrint(int n) {
	if (n == 0)
		return;

	cout << n << ' ';
	recursionPrint(n - 1);
}

File: year2020/month10/day1002/ms_1002.cs
using System;

namespace algorithm_70
{
    class ms_1002
    {
        static void RecursionPrint(int n) 
        {
            if(n == 0)
                return;
            
            Console.Write($"{n} ");
            RecursionPrint(n - 1);
        }

        static void Main(string[] args)
        {
            int n = Convert.ToInt32(Console.ReadLine());

            if(0 < n && n <= 50)
                RecursionPrint(n);
        }
    }
}
